### ISSUE TITLE
PHYDEV should not be mandatory for ovswitch

### ISSUE THREAD

**Issue by baby-gnu** [OpenNebula/one#2241]
# Bug Report
## Version of OpenNebula

- [ ] 5.2.2
- [ ] 5.4.0
- [ ] 5.4.1
- [ ] 5.4.2
- [ ] 5.4.3
- [ ] 5.4.4
- [ ] 5.4.5
- [ ] 5.4.6
- [ ] 5.4.7
- [ ] 5.4.8
- [ ] 5.4.9
- [ ] 5.4.10
- [ ] 5.4.11
- [ ] 5.4.12
- [ ] 5.4.13
- [ ] 5.4.14
- [ ] 5.4.15
- [x] Development build

## Component

- [ ] Authorization (LDAP, x509 certs...)
- [ ] Command Line Interface (CLI)
- [ ] Contextualization
- [ ] Documentation
- [ ] Federation and HA
- [ ] Host, Clusters and Monitorization
- [ ] KVM
- [x] Networking
- [ ] Orchestration (OpenNebula Flow)
- [ ] Packages
- [ ] Scheduler
- [ ] Storage & Images
- [ ] Sunstone
- [ ] Upgrades
- [ ] User, Groups, VDCs and ACL
- [ ] vCenter

## Description
An incompatible change is made to `ovswitch` driver : the `PHYDEV` attribute is now mandatory.

Here is the vnet template I used for ONE 5.4:

```
NAME = "CR_aca"
VN_MAD = ovswitch
BRIDGE = vswitch
AR=[
  TYPE = "ETHER",
  SIZE = "254"
]
```

### Expected Behavior

I can create completely isolated vnet without the `PHYDEV` attribute.

### Actual Behavior

I can't create the vnet with the following error:

```
Fri Jun 29 14:58:24 2018 [Z0][ReM][D]: Req:8608 UID:0 one.vn.allocate invoked , "NAME = "CR_aca"
VN_M...", -1
Fri Jun 29 14:58:24 2018 [Z0][VNM][E]: PHYDEV is mandatory for driver ovswitch
Fri Jun 29 14:58:24 2018 [Z0][ReM][E]: Req:8608 UID:0 one.vn.allocate result FAILURE [one.vn.allocate] PHYDEV is mandatory for driver ovswitch
```

## How to reproduce

1.
```
cat > /tmp/vnet.tmpl <<EOF
NAME = "CR_aca"
VN_MAD = ovswitch
BRIDGE = vswitch
AR=[
  TYPE = "ETHER",
  SIZE = "254"
]
EOF
```
2.
```
onevnet create /tmp/vnet.tmpl
```

# Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by baby-gnu** [comment]
I found #1807 and I'm more confused.

I think the `BRIDGE` is mandatory since to create the network interface libvirt use something like:

```
ovs-vsctl add-port <BRIDGE> one-<VMID>-<INTERFACEID>
```

But the `PHYDEV` should not.

Regards.

**Comment by baby-gnu** [comment]
Plus the documentation says it's optional for ovswitch:

* http://docs.opennebula.org/5.5/deployment/open_cloud_networking_setup/openvswitch.html#ovswitch-net
* http://docs.opennebula.org/5.5/operation/references/vnet_template.html#physical-network-attributes

Regards.

**Comment by rsmontero** [comment]
Fix for 5.6 oned and Sunstone.
